src/stirling/source_connectors/socket_tracer/tls_table.h

```diff
@@ -37,18 +37,22 @@ static constexpr DataElement kTLSElements[] = {
         canonical_data_elements::kLocalAddr,
         canonical_data_elements::kLocalPort,
         canonical_data_elements::kTraceRole,
-        {"req_type", "The type of request from the TLS record (Client/ServerHello, etc.)",
+        {"content_type", "The content type of the TLS record (e.g. handshake, alert, heartbeat, etc)",
```

Comment:
nit/subjective: Feel like the original `req_type` or something like `message_type` might be a better name, but I'll let you make the final call.

Part of the rationale is that there's a `req_body`, so `req_type`/`req_body` seem to pair up better.

Reply:
I was originally thinking that `req_type` could be confusing since a TLS 1.2 handshake (content type -- name per TLS spec) results in 2 round trips and 2 requests (ClientHello/ServerHello exchange followed by key exchange and cipher spec messages).

After thinking about it more, I've renamed it to `req_type` in 256b76e877cb07f2fa1424e9b71040f3b89b57e0 for consistency with our other tables.